src/Mod/Path/Path/Post/UtilsArguments.py

```diff
@@ -447,6 +449,12 @@ def init_shared_values(values):
     #
     values["OUTPUT_TOOL_CHANGE"] = True
     #
+    # This dictionary/hash holds the functions that are used
+    # to process the G-code parameter values
+    #
+    values["PARAMETER_FUNCTIONS"] = {}
```

Comment:
Would it make sense to call them `PARAMETER_VALUE_FORMATTERS` instead of `PARAMETER_FUNCTIONS`? To give a hint of what the functions do.

Likewise:
* `init_parameter_value_formatters(...)`
* `default_axis_value_formatter`
* ...

Should probably not be changed in this PR. Just a thought for the future.